handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: restart kibana
+  service: name=kibana state=restarted
```

Comment:
Please don't write all yml in one line. Change this everywhere.

Reply:
fixed